config.default.json

```diff
@@ -18,5 +18,70 @@
   "motion": {
      "enabled": false,
      "pin": "GPIO26"
-  }
-}
\ No newline at end of file
+  },
+  "speech": {
+    "keyFilename": "./keyfile.json",
+    "hotwords": [
+      {
+        "keyword": "smart mirror",
+        "model": "smart_mirror.pmdl"
+      }
+    ],
+    "sensitivity": "0.5",
+    "continuous": false
+  },
+    "geoPosition": {
+       "latitude": 41.547188,
+       "longitude": -87.72968600000002
+    },
```

Comment:
I'm Telling you it's killing us in support we need to at least do it this way for now... I'm looking at doing it locally in each plugin folder in the config.schema.json file... but it's still in testing this will solve at least 20% of the issues I'm dealing with during the day